x-pack/plugins/secops/public/components/timeline/body/helpers.ts

```diff
@@ -18,7 +18,7 @@ export const omitTypenameAndEmpty = (k: string, v: any): any | undefined =>
 
 export const stringifyEvent = (ecs: Ecs): string => JSON.stringify(ecs, omitTypenameAndEmpty, 2);
 
-export const eventHasNotes = (notes: Note[]): boolean => !isEmpty(notes);
+export const eventHasNotes = (notes: Note[] | string[]): boolean => !isEmpty(notes);
```

Comment:
Looks like you can remove the `Note[]` type if you're not intending it for future use. The only remaining usage is in this test: [secops/public/components/timeline/body/helpers.test.ts](https://github.com/xavierm/kibana/blob/f05bf082d9eb978ae9ccb7a80b8c86ae7d93743e/x-pack/plugins/secops/public/components/timeline/body/helpers.test.ts#L193)

Reply:
I would prefer the `Note[]` type over the `string[]`